wurst/util/Colors.wurst

```diff
@@ -162,3 +210,16 @@ init
 	assertTrue(mixed == colorA(128,128,128,128))
 
 	assertTrue(fromHexString("33ccff44") == colorA(204, 255, 68, 51))
+
+	assertTrue(color(255, 2, 2).getName() == "red")
```

Comment:
should use `color(255, 2, 2).getName().assertEquals("red")
This way the test will give proper feedback on failing and not just true/false.